iOS Example/iOS Example/ExampleTweaks.swift

```diff
@@ -27,7 +27,8 @@ public struct ExampleTweaks: TweakLibraryType {
 	public static let colorText1 = Tweak("Text", "Color", "text-1", UIColor.black)
 	public static let colorText2 = Tweak("Text", "Color", "text-2", UIColor(hue: 213/255, saturation: 0.07, brightness: 0.58, alpha: 1))
 
-    public static let title = Tweak<StringOption>.stringList("Text", "Text", "Title", options: ["SwiftTweaks", "Welcome!", "Example"])
+	public static let title = Tweak<StringOption>.stringList("Text", "Text", "Title", options: ["SwiftTweaks", "Welcome!", "Example"])
```

Comment:
Maybe let’s do `Title (StringList)` and `Subtitle (String)` for demo purposes! (I’ll do this in a follow-up diff)